modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/dnd_table/Cell.tsx

```diff
@@ -53,6 +53,15 @@ const Cell = ({
 		if (columnName && heading && !isFixed && cellRef.current) {
 			const boundingClientRect = cellRef.current.getBoundingClientRect();
 
+			const MINIMUM_COLUMN_WIDTH = 140;
```

Comment:
We typically place capitalized constants in the root context on top of the file.

Reply:
Let me know if there is anything else needed. Thank you!